api/v1beta1/lokistack_types.go

```diff
@@ -631,7 +631,7 @@ type LokiStackStatus struct {
 
 // LokiStack is the Schema for the lokistacks API
 //
-// +operator-sdk:csv:customresourcedefinitions:displayName="LokiStack",resources={{Deployment,v1},{StatefulSet,v1},{ConfigMap,v1},{Service,v1},{PersistentVolumeClaims,v1},{ServiceMonitor,v1}}
+// +operator-sdk:csv:customresourcedefinitions:displayName="LokiStack",resources={{Deployment,v1},{DaemonSet,v1},{StatefulSet,v1},{ConfigMap,v1},{Service,v1},{PersistentVolumeClaims,v1},{ServiceMonitor,v1}}
```

Comment:
This shouldn't be here, as per adding here only resources that are owned by LokiStack and reconciled by the operator.